solutions/swb-reference/src/config/example.yaml

```diff
@@ -20,7 +20,7 @@
 # If you want internet-facing put value `true` else `false`
 #albInternetFacing: true
 
-#hostZoneId: ''
+#hostedZoneId: ''
```

Comment:
This carried over from https://github.com/aws-solutions/solution-spark-on-aws/pull/798

Going to leave it since that should merge into develop soon.